.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: triage
+assignees: xgerman
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```

Comment:
Our steps should be:
1. Install Document DB Operator following `...`
2. Deploy a new Document DB Cluster using the manifest `...`
3. Connect to the Document DB Cluster using `...`
4. Test the behavior `...`